### PR TITLE
[matrix] 2nd round of corrections

### PR DESCRIPTION
**Pull request by jdewasseige**

### CODE REVIEW

File: src/q7/matrix-INMA2380/exercises/ch3.tex [hunk 1]
```diff
@@ -174,9 +175,9 @@ \section{Eigenvalues, eigenvectors and similitude}
     & = \fdif{I}{t} + \sum_{i=1}^\infty \frac{A^i}{i!} \fdif{t^i}{t}\\
     & = \sum_{i=1}^\infty \frac{iA^it^{i-1}}{i!}\\
     & = A\sum_{i=1}^\infty \frac{(At)^{i-1}}{(i-1)!}\\
-    & = A\sum_{i=0}^\infty \frac{A^i}{i!}\\
+    & = A \left(\sum_{i=0}^\infty \frac{A^i}{i!}\right) = A e^{At}\\
```
Comment (blegat) [inline src/q7/matrix-INMA2380/exercises/ch3.tex]: \sum_{i=0}^\infty \frac{A^i}{i!} = e^A mais pas e^{At} il me semble

Comment (jdewasseige) [reply src/q7/matrix-INMA2380/exercises/ch3.tex]: non c'est bien e^{At} mais je n'avais pas vu que le `t` de la ligne précédente avait disparu

File: src/q7/matrix-INMA2380/exercises/ch2.tex [hunk 2]
```diff
@@ -1,17 +1,38 @@
 \section{Linear system}
-\exo{0}
-Verify that
-$\mathcal{S}_1 \cap \mathcal{S}_2$,
-$\mathcal{S}_1 + \mathcal{S}_2$,
-$\Ker A$, $\Ima A$ and $A \mathcal{S}$
-are linear subspaced.
-\nosolution
+\exo{1}
```
Comment (blegat) [inline src/q7/matrix-INMA2380/exercises/ch2.tex]: I don't remember why I used `0` as exercise number and included the statement. Any idea why and why it needs to change ?

File: src/q7/matrix-INMA2380/exercises/ch1.tex [hunk 3]
```diff
@@ -268,16 +263,25 @@ \section{Matrix' algebra}
...
-  that they commute to every element of the base of $\mathcal{R}^{n \times n}$.
-  The canonical base is $e_ie_i^T$ for $i \in \{1, \ldots, n\}$ where
-  $e_k$ is a column vector with 1 at $k$ and 0 elsewhere.
-  We can see that $Ae_ie_i^T = A_{:i}e_i^T$ and $e_ie_i^TA = e_iA_{i:}$.
-
-  If $A_{:i}e_i^T = e_iA_{i:}$, the LHS imposes that only the $i$th column is non-zero
-  and the RHS imposes that only the $i$th line is non-zero.
-  We therefore need $A_{ki} = 0$ for $k \neq i$ and $A_{ik} = 0$ for $k \neq i$.
-  Since this must hold for every $i \in \{1, \ldots, n\}$, $A$ must be diagonal.
+  Let $B$ a square matrix of dimension $n$ commuting with $A = \diag\{a_1,\dots,a_n\}$,
```
Comment (blegat) [inline src/q7/matrix-INMA2380/exercises/ch1.tex]: Why not keeping both solutions ?

Comment (jdewasseige) [reply src/q7/matrix-INMA2380/exercises/ch1.tex]: I first thought the old solution was a copy paste of the previous one but just realised it's not. The question is asking to *prove that a matrix commuting with a diagonal one is itself diagonal*, so we could replace the first sentence by "Asking that it commutes with a diagonal matrix is equivalent to asking that it commutes to every element of the base $e_ie_i^T$ for $i \in \{1, \ldots, n\}$.", and keep both, what do you think?

File: src/q7/matrix-INMA2380/exercises/ch2.tex (hunk 2 above)
Comment (blegat) [reply src/q7/matrix-INMA2380/exercises/ch2.tex]: Ahah ok makes sense :-D

File: src/q7/matrix-INMA2380/exercises/ch1.tex (hunk 3 above)
Comment (blegat) [reply src/q7/matrix-INMA2380/exercises/ch1.tex]: Good idea. "base of $\mathcal{R}^{n \times n}$" is indeed wrong in the old one, it should be diagonal matrices

Comment (jdewasseige) [reply src/q7/matrix-INMA2380/exercises/ch1.tex]: @blegat done in 100ef72

**Pull request merged**